tests/integration/conftest.py

```diff
@@ -34,8 +34,8 @@ def get_engine(backend) -> sa.engine.Engine:
         account = os.environ.get("SNOWFLAKE_ACCOUNT", "")
         connection_string = f"snowflake://{user}:{password}@{account}/datajudge/DBO?warehouse=datajudge&role=accountadmin"
     elif "bigquery" in backend:
-        gcp_project = os.environ.get("GOOGLE_CLOUD_PROJECT", "scratch-361908")
-        connection_string = f"bigquery://{gcp_project}"
+        # gcp_project = os.environ.get("GOOGLE_CLOUD_PROJECT", "scratch-361908")

```

Comment:
Do we want to keep this?

Reply:
It's not needed. 

Before merging this, I want to see how much the time would improve if we use `pytest-xdist` to parallelize the test execution in BigQuery (maybe helps with Snowflake too).